examples/variables.tf

```diff
@@ -37,3 +37,24 @@ variable "public_ip_displayname" {
   description = "Display name for public IP."
   default     = "public-ip-test"
 }
+
+
+variable host_instance_type {
+  description = "The instance type for the ESX hosts in the primary cluster of the SDDC"
+  default     = "I3_METAL"
+}
+
+variable storage_capacity {
+  description = "The storage capacity value to be requested for the sddc primary cluster, in GiBs. If provided, instead of using the direct-attached storage, a capacity value amount of seperable storage will be used. Possible values for R5 metal are 15TB, 20TB, 25TB ,30TB, 35TB."
```

Comment:
I had reached out to the VMC team since this information is from their documentation. They said that the description says GiB but the the API that actually returns the information ie the capacity value -> the code returns TB for simplicity